lib/Services/api.dart

```diff
@@ -0,0 +1,74 @@
+import 'dart:convert';
+import 'dart:io';
+import 'package:basic_authentication_flutter_challenge/models/person.dart';
+import 'package:http/http.dart' as http;
+
+class Authentication {
+  static String accessToken = '';
+  static String userId = '';
+  static Person person = Person();
+  static final List<Person> teamMates = [];
```

Comment:
`teamMates` variable should not be inside `Authentication` class as it is not related to authentication capability. 
Either create a different class that holds this variable, or just define this variable inside `users_screen.dart` file.